RESTore/ExecutionContext.cs

```diff
@@ -425,9 +432,10 @@ public async Task MapCall()
         /// <returns>A task representing the executed call.</returns>
         private async Task<TimedResponse> ExecuteCall()
         {
+            HttpResponseMessage response = null;
```

Comment:
do we need to set response as null here? Cant we just have HttpResponseMessage response = await _httpClient.SendAsync(BuildRequest()); ??? I cant see it being used outside of the scope of this

Reply:
Changed on the performance branch, I believe